### DISCUSSION TITLE
2024/07/10 Co-working Meeting

### DISCUSSION THREAD

**Discussion by astroDimitrios** [carpentries-incubator/intro-to-modern-fortran#23] [Meetings]
## Notes:

- We discussed how to use the Workbench and got mostly the way there setting everyone up with RStudio to build the lesson
- We have opened a new issue #24 for people to add setup instructions for different IDEs
- We assigned the first two episodes for people to start adding content to

Links:

- The course outline #6 
- The [Workbench documentation](https://carpentries.github.io/sandpaper-docs/) and set up instructions
- The [Workbench component guide](https://carpentries.github.io/sandpaper-docs/component-guide.html)

## Sign In

Please sign in below so we have a record of who came!

- Dimitrios Theodorakis, (he/him) Met Office
- Yasel Quintero (she/her) TU Delft
- Luke Abraham, National Centre for Atmospheric Science and University of Cambridge
- Declan Valters (he/him), EPCC

**Comment by astroDimitrios** [comment]
Dimitrios Theodorakis, (he/him) Met Office

**Reply by theabro** [reply]
Luke Abraham, National Centre for Atmospheric Science and University of Cambridge

**Comment by dvalters** [comment]
Declan Valters (he/him), EPCC
